libtransmission/peer-mgr.cc

```diff
@@ -2578,7 +2578,7 @@ void get_peer_candidates(size_t global_peer_limit, tr_torrents& torrents, tr_pee
             continue;
         }
 
-        /* if everyone in the swarm is seeds and pex is disabled,
+        /* if everyone in the swarm are seeds and pex is disabled,
```

Comment:
this is generally incorrect grammar.  "everyone is" is correct usage.  there are exceptions particularly with counterfactuals and emphasize on "group" vs "each".

Reply:
57ddf6b Oops.